tokio-timer/src/interval.rs

```diff
@@ -29,6 +31,19 @@ impl Interval {
         Interval::new_with_delay(Delay::new(at), duration)
     }
 
+    /// Creates new `Interval` that yields with interval of `duration`.
+    ///
+    /// The function is shortcut for `Interval::new(Instant::now() + duration, duration)`.
+    ///
+    /// The `duration` argument must be a non-zero duration.
+    ///
+    /// # Panics
+    ///
+    /// This function panics if `duration` is zero.
+    pub fn interval(duration: Duration) -> Interval {
+        Interval::new(Instant::now() + duration, duration)
```

Comment:
Instead of `Instant::now()` could you use [`clock::now()`](https://docs.rs/tokio-timer/0.2.4/tokio_timer/clock/fn.now.html).

Reply:
Oh of course, i forgot that tokio has own clock settings